### ISSUE TITLE
Suggestion: Remove less-than-tasteful RMS quote

### ISSUE THREAD

**Issue by khionu** [meh/amnesia#88]
The README includes a quote from Stallman that is a bit off-putting. While I'm not one to discourage advocacy of issues, even in OSS projects, I think that, at minimum, there are other quotes by Stallman that could serve here, if I'm reading your intent correctly.

**Comment by MrTortoise** [comment]
Personally, I don't know who it's a quote from ... but I was sat thinking 'wtf, why is this being used as an example? What kind of deranged crazy people would I be supporting if I recommended this library at work ... what would others think if they see me recommending libraries where people have quotes like that in them?'

**Comment by khionu** [comment]
Yeah... Including the quote is meant to raise awareness of certain controversial things that Richard Stallman has said. The quote is as he said it. I don't interpret the inclusion as a sign of support of the statement, but it does have its problems being used in this position.

**Comment by jmerriweather** [comment]
Is it to raise awareness or be in support of said comments? Because I can't tell, I'd imagine a lot of other people can't tell either.

If you want to raise awareness of certain controversial comments, use twitter

**Comment by khionu** [comment]
Benefit of the doubt. Either way, we agree it shouldn't be here.
